### DISCUSSION TITLE
Pycryptodomex

### DISCUSSION THREAD

**Discussion by TheKnight-Dev** [flet-dev/flet#4325] [Packages] [answered]
Need a recipe for the `pycryptodomex `package as it is needed by `yt-dlp `.

Thanks.

**Comment by FeodorFitsner** [comment, marked as answer]
`pycryptodomex` and `pycryptodome` packages are now available for iOS and Android.

**Reply by mingoogle** [reply]
hello,
I’ve been learning to use Flet more effectively, and I truly appreciate your support.

I’m currently working on a project using the Flutter_Counter_Clone app (ft.app(target=main, view=ft.AppView.WEB_BROWSER)), and I’ve noticed that Flet doesn’t seem to work in the web environment. I was wondering if it might be possible to use pycryptodomex in a web browser environment?

Thank you so much for your time and assistance.

**Reply by FeodorFitsner** [reply]
`pycryptodome` (not `x`) is available in pyodide: https://pyodide.org/en/stable/usage/packages-in-pyodide.html

**Reply by TheKnight-Dev** [reply]
> `pycryptodomex` and `pycryptodome` packages are now available for iOS and Android.

Much appreciated!
